data/campaigns/Winds_of_Fate/scenarios/05_Threshold.cfg

```diff
@@ -865,7 +715,7 @@ We would be wise to secure all settlements before then."
 
 I have found anoþer Aquadyne. It is here, deep beneaþ Þe Þree Sisters. It has only been þree years since Haldric destroyed þe elf gosts þat were sapping it and already þere is a mælstrom formed off þe norþ shore. Just like þose mælstroms Seeker Perþ discovered west of Alduin before a Wisp chased away his ship.
```

Comment:
I somehow missed this the first time, but do we know if these characters can be displayed on all systems? Are there some that they will just be displayed as boxes on?

Reply:
These characters (upper and lower case "thorn" from the runic alphabet) are no longer part of English but are still used throughout modern day Icelandic. So my guess is any internationally compliant system must support it?

Reply:
I guess if they display as boxes, someone will let us know. But I'm leaving this unresolved for now since there's a chance someone else may come review before we merge who knows one way or another.

Reply:
It is to give the impression that the text has been incompletely cyphered by a mage of the Silver Order, and may also be an older dialect used for formal writing within the Order. Basically, it is a small word puzzle for players to decypher.

Sounds like the characters will work on Mac then, and I can confirm it works on Windows 10. So that just leaves Linux.

Reply:
Ultimately whether they work is mostly up to the font used, which if I'm not mistaken is controlled by Wesnoth. So I wouldn't expect this to differ by OS. The thorn character _might_ have an issue appearing in some translations that use completely different character sets which assign a different font, but those translations should've eliminated it in the translated text anyway.

I'm also think the po note for this string is wrong. Explaining the thorn isn't bad but isn't the most important detail. You should somehow incorporate what you said in the previous point into the note. That will allow translators to work out their own way of giving a similar impression.

Reply:
That is a good point. Updated the po note with this information.